app/controllers/outputs_controller.rb

```diff
@@ -0,0 +1,26 @@
+class OutputsController < ApplicationController
```

Comment:
I'm not sure about this name, `OutputsController` seems quite ambiguous. What about `ReportsController`?